### PR TITLE
Import ripemd160

### LINKED ISSUE #4235: "keybase pgp import -i seckey.asc" results in error
I'm trying to get set up with some existing gnupg keys. Unfortunately I get an error:

```
$ keybase pgp import -i seckey.asc 
▶ ERROR openpgp: unsupported feature: hash not available: 9
```

### PR DESCRIPTION
**Pull request by mlsteele**
Fixes #4235 

r? @maxtaco

### CODE REVIEW

File: go/libkb/pgp_key.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/keybase/go-crypto/openpgp/armor"
 	"github.com/keybase/go-crypto/openpgp/packet"
 	jsonw "github.com/keybase/go-jsonw"
+	_ "golang.org/x/crypto/ripemd160" // imported so that crypto supports ripemd160
```
Comment (maxtaco) [inline go/libkb/pgp_key.go]: can you say `// imported so that keybase/go-crypto/openpgp supports ripemd160`?

**Comment by maxtaco** [comment]
Need to vendor the library too to fix C/I

**Comment by mlsteele** [comment]
changed comment and vendored

**Comment by maxtaco** [comment]
You might have picked up some commits you didn't intend above?

**Comment by mlsteele** [comment]
whoops. Now should be good.

**Comment by maxtaco** [comment]
👍 ok great!

**Pull request merged**